### PR TITLE
slide: Update the uftrace.html with some enhancements

### PR DESCRIPTION
**Pull request by ClaudiaJKang**
This PR updates some points in [uftrace.github.io](https://uftrace.github.io). 

- #### Enhance pages about Display Function Arguments (#583)
    - A. Minor fixes about return values option page.

      
    
    - B. Replace some page's header about "return values" to "arguments" in Display Function Arguments pages.

      

Signed-off-by: Claudia J. Kang <claudiajkang@gmail.com>

### CODE REVIEW

**Comment by ClaudiaJKang** [comment]
@honggyukim I replace `hello.c` to `fibonacci.c` about  example for type setting. How do you think?

**Comment by honggyukim** [comment]
Hi @ClaudiaJKang, Thanks a lot for fixing the document, but please see the inline comments.

**Comment by honggyukim** [comment]
@namhyung I will. Thanks!

**Comment by ClaudiaJKang** [comment]
@honggyukim Sorry for late response. I fixed!

**Comment by ClaudiaJKang** [comment]
Ping?

**Comment by honggyukim** [comment]
Sorry for the delay. I made a few modifications based on your patch. Can you include this patch and update it on your branch? [0001-slide-Add-finstrument-functions-option-to-foobar.c.patch.txt](https://github.com/namhyung/uftrace/files/2614039/0001-slide-Add-finstrument-functions-option-to-foobar.c.patch.txt)

You can see the change in https://uftrace.github.io/slide/#12. Thanks for patience!

File: doc/uftrace.html [hunk 1]
```diff
@@ -1308,42 +1308,128 @@
...
+       0.883 us [31321] |       } `= 0x2`; /* fib */
+       0.125 us [31321] |       fib(2) `= 0x1`;
+       1.483 us [31321] |     } `= 0x3`; /* fib */
+                [31321] |     fib(3) {
+       0.125 us [31321] |       fib(2) `= 0x1`;
+       0.125 us [31321] |       fib(1) `= 0x1`;
+       0.774 us [31321] |     } `= 0x2`; /* fib */
+       2.716 us [31321] |   } `= 0x5`; /* fib */
+       4.382 us [31321] |   printf("fib(%d) = %d\n");
+       9.456 us [31321] | } /* main */
```
[the remark this answers is not in the document]
Comment (honggyukim) [reply doc/uftrace.html]: I was thinking about this change but it'd be better to keep the original one to simply show the usage in a short example. Sorry about that.

**Comment by honggyukim** [comment]
@ClaudiaJKang You can simply apply the attached patch using `git am` command.

**Comment by ClaudiaJKang** [comment]
@honggyukim I updated! Thank you!!

**Comment by honggyukim** [comment]
@ClaudiaJKang Thanks for your work!

**Pull request merged**